File: app/src/main/java/com/example/rdpocketpal2/util/StethoInitializer.java
package com.example.rdpocketpal2.util;

import android.content.Context;
import android.content.pm.ApplicationInfo;

import com.example.rdpocketpal2.BuildConfig;
import com.facebook.stetho.Stetho;

public class StethoInitializer {
    private final Context mContext;

    public StethoInitializer(Context context) {
        mContext = context;
    }

    public void initialize() {
        // initialize Stetho only if running a debug build
        if ((mContext.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE) != 0) {
            Stetho.initializeWithDefaults(mContext);
        }
    }
}

File: app/src/main/java/com/octrobi/rdpocketpal/util/StethoInitializer.kt
package com.octrobi.rdpocketpal.util

import android.content.Context
import android.content.pm.ApplicationInfo
import com.facebook.stetho.Stetho

class StethoInitializer(private val mContext: Context) {

    fun initialize() {
        // initialize Stetho only if running a debug build
        if (mContext.applicationInfo.flags and ApplicationInfo.FLAG_DEBUGGABLE != 0) {
            Stetho.initializeWithDefaults(mContext)
        }
    }
}